assemblies/pentaho-cde-renderer/feature/src/main/feature/feature.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<features name="${project.artifactId}-${project.version}" xmlns="http://karaf.apache.org/xmlns/features/v1.0.0">
+
+  <feature name="foundry-cde-renderer" version="${project.version}">
+    <feature version="${pentaho.version}">pentaho-requirejs-osgi-manager</feature>
+    <feature version="${pentaho.version}">pentaho-webpackage</feature>
```

Comment:
You're hardcoding the `pentaho-requirejs-osgi-manager` feature but not the `pentaho-webpackage` one. Where is the `pentaho-webpackage` feature coming from?
If its comming from this artifact: https://github.com/webdetails/cde/pull/819/files/a2a614229bf7999b5c72e0ea645cde3ba5bde601#diff-28f353f38a4349db1ae5f8a0d6373e32R41
, then just add the dependency to the pom and remove the hardcoded feature from here.